Not able to compile the project as given in the README.md

### Description
I tried the 'npm start' command from the directory of master.

### Expected Behavior
Should compile and give output that the project is compiled successfully.

### Actual Behavior
Got the errors as follows:
E:\videogular2-master>npm start

> videogular2@0.0.0-semantically-released start E:\videogular2-master
> parallelshell 'npm run watch:ts' 'npm run start:coverage-server'

'parallelshell' is not recognized as an internal or external command,
operable program or batch file.

npm ERR! Windows_NT 10.0.14393
npm ERR! argv "C:\\Program Files (x86)\\nodejs\\node.exe" "C:\\Users\\PreciseT9\\AppData\\Roaming\\npm\\node_modules\\npm\\bin\\npm-cli.js" "start"
npm ERR! node v6.9.1
npm ERR! npm  v3.10.9
npm ERR! code ELIFECYCLE
npm ERR! videogular2@0.0.0-semantically-released start: `parallelshell 'npm run watch:ts' 'npm run start:coverage-server'`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the videogular2@0.0.0-semantically-released start script 'parallelshell 'npm run watch:ts' 'npm run start:coverage-server''.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the videogular2 package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     parallelshell 'npm run watch:ts' 'npm run start:coverage-server'
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs videogular2
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls videogular2
npm ERR! There is likely additional logging output above.
npm WARN Local package.json exists, but node_modules missing, did you mean to install?

npm ERR! Please include the following file with any support request:
npm ERR!     E:\videogular2-master\npm-debug.log

### Steps to Reproduce
Type 'npm start' in the root directory in command prompt

EDIT: I installed the package into Visual Studio and tried to create a basic HTML page with a single video, only to find out that there are too many errors in the package itself. I think this is not a release candidate.

--- a/package.json
+++ b/package.json
@@ -59,7 +59,8 @@
     "posttest": "remap-istanbul -i coverage/coverage-final.json -o coverage -t html",
     "report-coverage": "cat ./coverage/lcov.info | codecov",
     "semantic-release": "semantic-release pre && npm publish && semantic-release post",
-    "watch:ts": "watch 'npm run build' src"
+    "watch:ts": "watch 'npm run build' src",
+    "parallelshell": "parallelshell"
   },
   "engines": {
     "node": ">=0.8.0"

